src/components/pages/NotificationCenter/NotificationPager.tsx

```diff
@@ -0,0 +1,65 @@
+/********************************************************************************
+ * Copyright (c) 2021, 2023 BMW Group AG
+ * Copyright (c) 2021, 2023 Contributors to the Eclipse Foundation
```

Comment:
Could you please update the header to '2021, 2024' in every newly added file?
The template for the header still needs to get updated for the new year but I'd do that in a separate PR.

Reply:
Header updated